DQMOffline/Trigger/plugins/TopMonitor.h

```diff
@@ -1,250 +0,0 @@
-#ifndef DQMOffline_Trigger_TopMonitor_h
```

Comment:
@missirol  I don't understand why in this case you move everything from .h to .cc file while in the other plugins you don't

Reply:
it's a fair point, in principle it should be done (i had done it for `TopMonitor` extemporaneously, and this got included in this PR); for the other plugins, i was trying to stick to the changes i originally planned (the ones described in the PR)

Reply:
@missirol I don't have a preference so I let code experts to evaluate and decide but I believe there should be coherence across the other plugins